taskcluster/fenix_taskgraph/release_promotion.py

```diff
@@ -12,10 +15,8 @@
 from taskgraph.parameters import Parameters
 from taskgraph.util.taskgraph import find_decision_task, find_existing_tasks_from_previous_kinds
 
-from .parameters import resolve_release_type
-
 RELEASE_PROMOTION_PROJECTS = (
-    "https://github.com/JohanLorenzo/fenix",
```

Comment:
My fork may be needed in the future, but we don't need it to be whitelisted on the main repo.